app/__init__.py

```diff
@@ -30,5 +29,10 @@ def create_app(test_config=None):
     migrate.init_app(app, db)
 
     # Register Blueprints here
+    from .routes import tasks_bp
+    app.register_blueprint(tasks_bp)
+
+    from .routes import goals_bp
+    app.register_blueprint(goals_bp)
```

Comment:
To reduce redundancy, imports can be combined:

```suggestion
    from .routes import tasks_bp, goals_bp
    app.register_blueprint(tasks_bp)
    app.register_blueprint(goals_bp)
```